dart/lib/src/sentry_options.dart

```diff
@@ -136,6 +136,17 @@ class SentryOptions {
   /// example : ['sentry'] will include exception from 'package:sentry/sentry.dart'
   List<String> get inAppIncludes => List.unmodifiable(_inAppIncludes);
 
+  /// Configures whether stack trace frames are considered in app frames by default.
+  /// You can use this to essentially make [inAppIncludes] or [inAppExcludes]
+  /// an allow or deny list.
+  /// This value is only used if Sentry can not find the origin of the frame.
+  ///
+  /// - If [considerFramesInAppByDefault] is true you only need to maintain
+  /// [inAppExcludes].
+  /// - If [considerFramesInAppByDefault] is false you only need to maintain
+  /// [inAppIncludes].
+  bool considerFramesInAppByDefault = true;
```

Comment:
```suggestion
  bool considerInAppFramesByDefault = true;
```

looks like this reads better, wdyt?

Reply:
I don't think so but the docs are good so I'm fine with both names.